examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -3808,6 +3808,12 @@ bool PhysicsServerCommandProcessor::processCommand(const struct SharedMemoryComm
 									{
 										linkIndexB = mblB->m_link;
 										objectIndexB = mblB->m_multiBody->getUserIndex2();
+										if (
```

Comment:
Why is this filterneeded? Say the filter is for object 2 and object 3, then we should also allow object 3 and object 2 (the order doesn't matter). This check is already applied in the existing code.